.github/workflows/verify-pr-commit.yml

```diff
@@ -235,6 +235,10 @@ jobs:
     steps:
       - name: Check Out Repo
         uses: actions/checkout@v3
+      - name: Install Required Packages
+        run: |
+          sudo apt-get update
+          sudo apt install -y protobuf-compiler
```

Comment:
@demisx  can you check not sure if it is working, I see code cov failed due to protoc